index.js

```diff
@@ -19,9 +19,23 @@ const SETTINGS_SCHEMA = [
     title: 'Minimum interval',
     enumPicker: 'select',
     enumChoices: ['days', 'hours', 'minutes', 'seconds'],
-    description: 'If set to hours, the annotation will only be added if the interval is at least an hour. Note: Annotations are only generated when the page loads and do not update in real time.',
+    description: 'If set to hours, the annotation will only be shown if the interval is at least an hour.',
     default: 'hours',
   },
+  {
+    key: 'update-on-edit',
+    type: 'boolean',
+    title: 'Update on edit',
+    description: 'Dynamically updates annotations when the time is edited. Requires a page refresh to apply. Note: May have a noticeable performance impact on pages with many schedule/deadline entries.',
+    default: true,
+  },
+  {
+    key: 'update-interval',
+    type: 'number',
+    title: 'Update interval',
+    description: 'Update annotations at this interval (seconds). Any value below 1 disables this feature. Note: Updates have a performance cost. Setting the value lower than 600 (10 minutes) is not recommended.',
```

Comment:
I guess the users of this plugin will most often just need to take a glance at some dedicated page where tasks are collected (or a query) to check the tasks and the deadline. In this case, update is not needed at all actually. We don't need a timer in LogSeq :p

Reply:
Since I've actually rewritten the entire plugin at this point, it probably doesn't make much sense not to just create my own project. Especially since it seems like we have different opinions about what features should be included.

I will leave this PR open in case you want to use parts of it. You will need to make the modifications though.

Reply:
I'd like to keep this plugin minimal. You'd better create your own plugin to add more advanced features like timer. Anyway, thanks for your interest and understanding!

Reply:
Absolutely. Have a great day!